.travis.yml

```diff
@@ -11,6 +11,7 @@ matrix:
   fast_finish: true
 script:
   - npm run lint
+  - npm run prettier
```

Comment:
Doing this will obviously let the prettier check pass 😄 . Without committing the changes it really defeats the purpose. Run `npm run prettier` on your PR branch and commit the changes (if any)